bin/fetch_github_actions_quota.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+import logging
+
+from services.github_service import GithubService
+from services.kpi_service import KpiService
+from config.constants import ENTERPRISE, MINISTRY_OF_JUSTICE
+
+
+def get_environment_variables() -> str:
+    github_token = os.getenv("GH_TOKEN")
+    if not github_token:
+        raise ValueError(
+            "The env variable GH_TOKEN is empty or missing")
+
+    return github_token
+
+
+def fetch_gha_quota():
+    github_token = get_environment_variables()
+    if github_token is None:
```

Comment:
This can probably be removed since the functionality is covered by `get_environment_variables` 😁

Reply:
ups....